### PR TITLE
Changed readme and sample folders to have app-manifest name

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/app-auth/nodejs/README.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ This sample demonstrates authentication in Microsoft Teams using bot and tab.
...
 > Azure Bot Service to use the authentication functionality provided by Azure Bot Service.
 
-4. Create an app manifest. Navigate to the file, manifest/manifest.json - Change:
+4. Create an app manifest. Navigate to the file, appManifest/manifest.json - Change:
     1. <<REGISTERED_BOT_ID>> (there are 3) change to your registered bot's app ID
     2. <<BASE_URI_DOMAIN>> (there are 5) change to your https endpoint from ngrok excluding the "https://" part
-    **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `app-auth\nodejs\Manifest_Hub` folder with the required values.
-    3. Zip up the contents of the `manifest` folder to create a `manifest.zip` or `Manifest_Hub` folder into a `Manifest_Hub.zip`. (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `app-auth\nodejs\appManifestHub` folder with the required values.
```
Comment (ChetanSharma-msft) [inline samples/app-auth/nodejs/README.md:66]: Please don't remove underscore  appManifest_Hub

File: samples/app-complete-auth/csharp/README.md [hunk 2]
```diff
@@ -155,9 +155,9 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
    - `{{Microsoft-App-Id}}` with Application id generated from Step 3
    - `{Base_URL_Domain}` - Your application's base url domain. E.g. for https://12345.ngrok-free.app the base url domain will be 12345.ngrok-free.app if you are using ngrok.
 
-**Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `app-complete-auth\csharp\AppCompleteAuth\Manifest_Hub` folder with the required values.
+**Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `app-complete-auth\csharp\AppCompleteAuth\AppManifestHub` folder with the required values.
```
Comment (ChetanSharma-msft) [inline samples/app-complete-auth/csharp/README.md:158]: Please don't remove underscore

File: samples/app-complete-sample/csharp/README.md [hunk 3]
```diff
@@ -113,17 +113,17 @@ NOTE: Teams does not work nor render things exactly like the Bot Emulator, but i
 
 7. Setup Manifest for Teams
 	- __*This step is specific to Teams.*__
-	    - **Edit** the `manifest.json` contained in the ./manifest or ./manifest_hub folder to replace your Microsoft App Id (that was created when you registered your app registration earlier) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
+	    - **Edit** the `manifest.json` contained in the ./appManifest or ./appManifestHub folder to replace your Microsoft App Id (that was created when you registered your app registration earlier) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```
Comment (ChetanSharma-msft) [inline samples/app-complete-sample/csharp/README.md:116]: Don' remove underscore.

File: samples/app-complete-sample/csharp/README.md [hunk 4]
```diff
@@ -113,17 +113,17 @@ NOTE: Teams does not work nor render things exactly like the Bot Emulator, but i
...
 
 	- Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
 	   - Go to Microsoft Teams. From the lower left corner, select Apps
 	   - From the lower left corner, choose Upload a custom App
-	   - Go to your project directory, the ./manifest folder, select the zip folder, and choose Open.
+	   - Go to your project directory, the ./appManifest folder, select the zip folder, and choose Open.
 	   - Select Add in the pop-up dialog box. Your app is uploaded to Teams.
    		
-**Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `/manifest_hub` folder with the required values.
+**Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `/appManifestHub` folder with the required values.
```
Comment (ChetanSharma-msft) [inline samples/app-complete-sample/csharp/README.md:126]: Don' remove underscore.

File: samples/app-localization/csharp/README.md [hunk 5]
```diff
@@ -75,12 +75,12 @@ Verify you have the right account for building Teams apps and install some recom
...
+    - **Edit** the `manifest.json` contained in the `AppManifest` or `AppManifestHub` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` and <<Azure Bot ID>> and for the contentUrl "<<1234.ngrok-free.app>>?culture={locale}" (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
     - **Provide ngrok Url** (turnnelling Url) in `manifest.json` for contentUrl in case of tabs and for messaging endpoint in case of bots if enabled. 
-    - **Zip** up the contents of the `Manifest` or `Manifest_hub` folder to create a `manifest.zip`
+    - **Zip** up the contents of the `AppManifest` or `AppManifestHub` folder to create a `manifest.zip`
```
Comment (ChetanSharma-msft) [inline samples/app-localization/csharp/README.md:80]: Don't remove underscore

File: samples/app-task-module/csharp/README.md [hunk 6]
```diff
@@ -91,15 +91,15 @@ Task modules build on the foundation of Microsoft Teams tabs: a task module is e
...
-    - **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `app-task-module\csharp\Microsoft.Teams.Samples.TaskModule.Web\Manifest_Hub` folder with the required values.
-    - **Zip** up the contents of the `Manifest` folder to create a `Manifest.zip` or `Manifest_Hub` folder into a `Manifest_Hub.zip`. (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    - **Note:** If you want to test your app across multi hub like: Outlook/Office.com, please update the `manifest.json` in the `app-task-module\csharp\Microsoft.Teams.Samples.TaskModule.Web\AppManifestHub` folder with the required values.
+    - **Zip** up the contents of the `AppManifest` folder to create a `Manifest.zip` or `AppManifestHub` folder into a `AppManifestHub.zip`. (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
 
 - Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
```
Comment (ChetanSharma-msft) [inline samples/app-task-module/csharp/README.md:99]: Check all the files changes for small and capital letters.
C# sample also you have used appManifest rather than AppManifest.
Also, for Manifest_Hub => AppManifest_Hub rather than AppManifestHub

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comments.

File: samples/app-auth/nodejs/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/app-auth/nodejs/README.md:72]: changed

File: samples/app-complete-sample/csharp/README.md (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/app-complete-sample/csharp/README.md]: changed

File: samples/app-complete-sample/csharp/README.md (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/app-complete-sample/csharp/README.md]: changed

File: samples/app-localization/csharp/README.md (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/app-localization/csharp/README.md]: changed

File: samples/app-task-module/csharp/README.md (hunk 6 above)
Comment (Prithvi-MSFT) [reply samples/app-task-module/csharp/README.md:105]: changed

**Review by Mohammed-MSFT — APPROVED** [review]

File: samples/account-linking/csharp/README.md [hunk 7]
```diff
@@ -77,7 +77,7 @@ For this example, we'll use the `590a2d6f8b31.ngrok-free.app` as our domain name
 Please follow the instructions on [creating an Azure AD application with Tab SSO](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso?tabs=dotnet#1-create-your-azure-ad-application). The fully qualified domain name will be the ngrok url from before.
 
 #### 2.1 Configure the app for v2 tokens
```
Comment (ChetanSharma-msft) [inline samples/account-linking/csharp/README.md:79]: @Prithvi-MSFT - Your PR changes are not consistent with below:
image


Somewhere you are using appManifestHub and other places you are using appManifest_Hub

Kindly update the PR changes properly.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comments

File: samples/app-task-module/csharp/README.md (hunk 6 above)
Comment (ChetanSharma-msft) [reply samples/app-task-module/csharp/README.md:105]: @Prithvi-MSFT - you did not change it at all the places.

File: samples/app-localization/csharp/README.md [hunk 8]
```diff
@@ -81,13 +81,13 @@ Verify you have the right account for building Teams apps and install some recom
...
-    - **Provide turnnelling Url** in `manifest.json` for contentUrl in case of tabs and for messaging endpoint in case of bots if enabled. 
-    - replace `{{domain-name}}` with base Url of your domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app` and if you are using dev tunnels then your domain will be like: `12345.devtunnels.ms`.
+    - **Edit** the `manifest.json` contained in the `Manifest` or `Manifest_hub` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` and <<Azure Bot ID>> and for the contentUrl "<<1234.ngrok-free.app>>?culture={locale}" (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```
Comment (ChetanSharma-msft) [inline samples/app-localization/csharp/README.md:84]: image

File: samples/bot-people-picker-adaptive-card/csharp/README.md [hunk 9]
```diff
@@ -82,13 +82,13 @@ In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/azure/
  
 5. Setup Manifest for Teams
 - __*This step is specific to Teams.*__
-    - **Edit** the `manifest.json` contained in the ./AppPackage folder to replace your Microsoft App Id (that was created when you registered your app registration earlier) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)   
-    - **Zip** up the contents of the `AppPackage` folder to create a `manifest.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    - **Edit** the `manifest.json` contained in the ./AppManifest folder to replace your Microsoft App Id (that was created when you registered your app registration earlier) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)   
+    - **Zip** up the contents of the `AppPaAppManifestckage` folder to create a `manifest.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
```
Comment (ChetanSharma-msft) [inline samples/bot-people-picker-adaptive-card/csharp/README.md:86]: Wrong name of folder.

File: samples/bot-proactive-messaging/csharp/README.md [hunk 10]
```diff
@@ -68,9 +68,9 @@ Two samples to highlight solutions to two challenges with building proactive mes
 - Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
 - configure the messaging endpoint in Azure bot with your application domain URL and append `/api/messages` like: `https://111xxx.ngrok-free.app/api/messages`
 
-3. Modify the `manifest.json` in the `/teamsAppManifest` folder and replace the `{{BOT-ID}}` with the id received while doing AAD app registration in previous steps and also update the `<<App Domain>>` with your application domain like ngrok domain: `111xxx.ngrok-free.app` or dev tunnels domain: `12345.devtunnels.ms` excluding http/https.
+3. Modify the `manifest.json` in the `/AppManifest` folder and replace the `{{BOT-ID}}` with the id received while doing AAD app registration in previous steps and also update the `<<App Domain>>` with your application domain like ngrok domain: `111xxx.ngrok-free.app` excluding http/https.
```
Comment (ChetanSharma-msft) [inline samples/bot-proactive-messaging/csharp/README.md:71]: Revert the change to include dev tunnel related.

File: samples/bot-sso-adaptivecard/csharp/BotSsoAdaptivecard/README.md [hunk 11]
```diff
@@ -149,6 +149,7 @@ Register your application with Azure AD
 **This step is specific to Teams.**
    - **Edit** the `manifest.json` contained in the  `TeamsAppManifest` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
    - **Edit** the `manifest.json` for `validDomains` and `{{domain-name}}` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app` and if you are using dev tunnels then your domain will be `12345.devtunnels.ms`.
+   - **Edit** he manifest.json for webApplicationInfo resource "api://botid-`<<YOUR-MICROSOFT-APP-ID>>`" with base Url of your domain. E.g. if you are using ngrok it would be https://1234.ngrok-free.app then your domain-name will be "api://botid-`<<YOUR-MICROSOFT-APP-ID>>`".
```
Comment (ChetanSharma-msft) [inline samples/bot-sso-adaptivecard/csharp/BotSsoAdaptivecard/README.md:152]: he ??

File: samples/bot-task-module/csharp/README.md [hunk 12]
```diff
@@ -87,7 +87,7 @@ the Teams service needs to call into the bot.
 
 1) __*This step is specific to Teams.*__
     - **Edit** the `manifest.json` contained in the `TeamsAppManifest` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
-    - **Edit** the `manifest.json` for `<<Domain-Name>>` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app` and if you are using dev tunnels then your domain will be like: `12345.devtunnels.ms`.
+    - **Edit** the `manifest.json` for `validDomains` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app` and if you are using dev tunnels then your domain will be like: `12345.devtunnels.ms`.
```
Comment (ChetanSharma-msft) [inline samples/bot-task-module/csharp/README.md:90]: It should be `<<Domain-Name>>` ?

File: samples/msgext-action/java/README.md [hunk 13]
```diff
@@ -110,7 +110,7 @@ the Teams service needs to call into the bot.
 3. Selecting the **FetchRoster** command from the Message command list. *Even though this action is being shown on the contextual menu, it's not implemented and is expected to fail.*
 ![FetchRoster](Images/FetchRoster.PNG)
 
-**Note:** Likewise you can try with all other configured commands in you manifest.
+**Note:** Likewise you can try with all other configured commands in you app manifest.
```
Comment (ChetanSharma-msft) [inline samples/msgext-action/java/README.md:113]: your

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/msgext-action/java/README.md (hunk 13 above)
Comment (Prithvi-MSFT) [reply samples/msgext-action/java/README.md:113]: added

File: samples/bot-sso-adaptivecard/csharp/BotSsoAdaptivecard/README.md (hunk 11 above)
Comment (Prithvi-MSFT) [reply samples/bot-sso-adaptivecard/csharp/BotSsoAdaptivecard/README.md:152]: changed

File: samples/bot-task-module/csharp/README.md (hunk 12 above)
Comment (Prithvi-MSFT) [reply samples/bot-task-module/csharp/README.md:90]: changed

File: samples/app-complete-sample/csharp/README.md [hunk 14]
```diff
@@ -46,7 +46,7 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
   ```bash
   dotnet --version
   ```
-- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/download) (For local environment testing) latest version (any other tunneling software can also be used)
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/app-complete-sample/csharp/README.md:49]: changed

File: samples/app-complete-sample/csharp/README.md [hunk 15]
```diff
@@ -60,22 +60,16 @@ NOTE: Teams does not work nor render things exactly like the Bot Emulator, but i
 	- Register a AAD aap registration in Azure portal.
 	- Also, register a bot with Azure Bot Service, following the instructions [here](https://docs.microsoft.com/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0).
 	- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
-	- While registering the bot, use `https://<your_tunnel_domain>/api/messages` as the messaging endpoint.
+	- While registering the bot, use `https://<your_ngrok_url>/api/messages` as the messaging endpoint.
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/app-complete-sample/csharp/README.md:63]: changed

File: samples/app-complete-sample/csharp/README.md [hunk 16]
```diff
@@ -60,22 +60,16 @@ NOTE: Teams does not work nor render things exactly like the Bot Emulator, but i
...
 
     > NOTE: When you create your app registration, you will create an App ID and App password - make sure you keep these for later.
 
-3. Run ngrok - point to port 3978
-
-   ```bash
-   ngrok http 3978 --host-header="localhost:3978"
-   ```  
-
-   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/app-complete-sample/csharp/README.md:73]: changed

File: samples/app-complete-sample/csharp/README.md [hunk 17]
```diff
@@ -119,17 +113,17 @@ NOTE: Teams does not work nor render things exactly like the Bot Emulator, but i
...
-	    - **Edit** the `manifest.json` for `validDomains` and replace `{{domain-name}}` with base Url of your domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app` and if you are using dev tunnels then your domain will be like: `12345.devtunnels.ms`.
-	    - **Zip** up the contents of the `manifest` or 'manifest_hub.json' folder to create a `manifest.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+	    - **Edit** the `manifest.json` contained in the ./AppManifest or ./AppManifest_Hub folder to replace your Microsoft App Id (that was created when you registered your app registration earlier) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
+	    - **Edit** the `manifest.json` for `validDomains` and replace `{{domain-name}}` with base Url of your domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app`.
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/app-complete-sample/csharp/README.md:117]: changed

File: samples/bot-people-picker-adaptive-card/csharp/README.md (hunk 9 above)
Comment (Prithvi-MSFT) [reply samples/bot-people-picker-adaptive-card/csharp/README.md:86]: changed

File: samples/bot-proactive-messaging/csharp/README.md (hunk 10 above)
Comment (Prithvi-MSFT) [reply samples/bot-proactive-messaging/csharp/README.md:71]: changed

File: samples/app-localization/csharp/README.md (hunk 8 above)
Comment (Prithvi-MSFT) [reply samples/app-localization/csharp/README.md:84]: changed

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comments

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6222803322
